Yosemite/Yosemite/Stores/AnnouncementsStore.swift

```diff
@@ -0,0 +1,108 @@
+import Networking
+import Storage
+
+/// Protocol for `AnnouncementsRemote` mainly used for mocking.
+///
+public protocol AnnouncementsRemoteProtocol {
+
+    func getFeatures(appId: String,
+                     appVersion: String,
+                     locale: String,
+                     completion: @escaping (Result<[WooCommerceFeature], Error>) -> Void)
+}
+
+// MARK: - AnnouncementsStore
+//
+public class AnnouncementsStore: Store {
+
+    typealias IsCached = Bool
+    typealias AppVersion = String
+    private let remote: AnnouncementsRemoteProtocol
+    private let fileStorage: FileStorage
+
+    public init(dispatcher: Dispatcher,
+                storageManager: StorageManagerType,
+                network: Network,
+                remote: AnnouncementsRemoteProtocol,
+                fileStorage: FileStorage) {
+        self.remote = remote
+        self.fileStorage = fileStorage
+        super.init(dispatcher: dispatcher, storageManager: storageManager, network: network)
+    }
+
+    private var appVersion: AppVersion { UserAgent.bundleShortVersion }
+
+    private lazy var featureAnnouncementsFileURL: URL! = {
+        let documents = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
+        return documents!.appendingPathComponent(Constants.featureAnnouncementsFileName)
+    }()
```

Comment:
We try to avoid using implicitly unwrapped optional.
Do you think you can model these without them?